django/db/backends/base/features.py

```diff
@@ -368,6 +368,9 @@ class BaseDatabaseFeatures:
     # Does the backend support unlimited character columns?
     supports_unlimited_charfield = False
 
+    # Does the backend require a GROUP BY clause with a HAVING clause?
```

Comment:
uh just realised this phrase is ambiguous…

originally I had `allows_having_without_group_by` but inverted it to make it read in the positive for the condition in `SQLCompiler`